languages/javascript/config.json

```
@@ -27,6 +27,12 @@
         "concepts": ["booleans-basic"],
         "prerequisites": []
       },
+      {
+        "slug": "arrays",
+        "uuid": "25cb0707-44f8-4800-b993-3fcb2b6d9f61",
+        "concepts": ["arrays-basic"],
```

We've recently [updated the Concepts naming rules](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts), which would make this concept be listed as:

```suggestion
        "concepts": ["arrays"],
```

@SleeplessByte Would you prefer to update this PR with the new rules or update them in a separate PR?

---

I've committed this suggested change.

---

Apparently it was changed back.

---

```suggestion
        "concepts": ["arrays"],
```